imgui-knobs.cpp

```diff
@@ -9,6 +9,46 @@
 
 namespace ImGuiKnobs {
     namespace detail {
+
+
+
+    //Re-maps a number from one range to another.
+    //Numbers outside the range are not clamped to 0 and 1.
+    template<typename T>
+    inline T Map(const T& aValue,const T& aIStart,const T& aIStop,const T& aOStart,const T& aOStop)
```

Comment:
`Map`  is only used inside `MapC`, can you inline it there?

Reply:
Yes, I copied it from code I was using. As I did not know if you wanted to merge I did not modified it.